src/components/action-bar/action-bar.tsx

```diff
@@ -75,6 +84,12 @@ export class ActionBar {
     @Prop({ reflect: true })
     public layout?: 'fullWidth' | 'floating';
 
+    @Prop({ reflect: true })
+    public actionBarIsShrunk = false;
```

Comment:
Why do we have this as a prop? Only to reflect it?

Reply:
I have changed it to `@State()`